Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -696,20 +707,9 @@ IPCCommandResult ES::ExportContentData(Context& context, const IOCtlVRequest& re
 
 ReturnCode ES::ExportContentEnd(Context& context, u32 content_fd)
 {
-  const auto iterator = context.title_export.contents.find(content_fd);
-  if (!context.title_export.valid || iterator == context.title_export.contents.end() ||
-      iterator->second.content.m_position != iterator->second.content.m_content.size)
-  {
+  if (!context.title_import.valid || !context.title_import.content.valid)
```

Comment:
Yes. The old code added too many checks that weren't actually done by ES. This is the [ExportContentEnd](https://gist.github.com/leoetlino/ef700cce50930e65e40b7b748cb0ad50) function for the reference. Checks that are specific to the export code (context validity checks) are done here, whereas the cfd bound check is done in CloseContent.

You may have noticed that we reuse the content related functions, whereas IOS's export code just uses the fd directly -- this is partly because we don't emulate internal fd tables currently (and it should be equivalent anyway).